instagram/src/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import dummyData from './dummy-data.js'
+import { PostContainer } from './components/PostContainer/PostContainer.js'
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+       DummyData: [],
```

Comment:
`DummyData` should be lowercase. There are only a few scenarios where you would want to use uppercase in JS (`class`, React components, etc.)